### PR TITLE
Add function documentation to room.lua

### PR DESCRIPTION
**Pull request by Alberth289346**

### CODE REVIEW

File: CorsixTH/Lua/room.lua [hunk 1]
```diff
@@ -66,6 +66,9 @@ function Room:initRoom(x, y, w, h, door, door2)
   self.humanoids_enroute = {--[[a set rather than a list]]}
 end
 
+--! Get the tile next to the door.
+--!param inside (bool) If set, get the tile inside the room, else get the tile outside.
+--!return x,y (tile coordniates) of the tile next to the door.
```
Comment (sadger) [inline CorsixTH/Lua/room.lua]: Typo s/coordniates/coordinates

File: CorsixTH/Lua/room.lua [hunk 2]
```diff
@@ -407,17 +421,23 @@ function Room:createDealtWithPatientCallback(humanoid)
   self.waiting_staff_member = humanoid
 end
 
--- Returns the current staff member. Can be overriden in rooms with multiples staff members to return the desired one.
+--! Get the current staff member.
+-- Can be overridden in rooms with multiples staff members to return the desired one.
```
Comment (sadger) [inline CorsixTH/Lua/room.lua]: multiples should be multiple if we are being picky - plural noun

File: CorsixTH/Lua/room.lua [hunk 3]
```diff
@@ -595,6 +615,9 @@ end
 
 local tile_factor = 10     -- how many tiles further are we willing to walk for 1 person fewer in the queue
 local readiness_bonus = 50 -- how many tiles further are we willing to walk if the room has all the required staff
+
+--! Score function to decide how desire-full a room is for a patient.
```
Comment (sadger) [inline CorsixTH/Lua/room.lua]: desire-full should be desirable

**Comment by Alberth289346** [comment]
All fixed

**Comment by sadger** [comment]
They look fine to me now. Thanks @Alberth289346

**Pull request merged**